### PR TITLE
Updated for deployment.

### PR DESCRIPTION
**Pull request by theofficetroll**
Will add in various services as they are deployed as well as default data going forward.

https://trello.com/c/iKj3U9NS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,16 @@ To Start Servers:
 Proxy : npm run server-dev
 freddie-individual-item-details : npm run server
 kimarie-bonuses : npm run start
+max-descriptions : npm run server-dev
```
Comment (gektin) [inline README.md:6]: I would use `npm run start` for my service.

File: server/index.js [hunk 2]
```diff
@@ -11,72 +11,46 @@ app.listen(PORT, () => {
...
+//     let pieces = req.url.split('?');
+//     let pathname = pieces[0];
+//     let queryString = pieces[1];
+//     if (!queryString) {
+//       return '';
+//     } else {
+//       return `${pathname}?${queryString}`;
+//     }
+//   }
+// }));
```
Comment (gektin) [inline server/index.js:50]: All this is preserved in your commit history. There's no need to keep this commented out code here unless you think you'll need to uncomment it in the near term.

File: README.md [hunk 3]
```diff
@@ -3,15 +3,16 @@ To Start Servers:
...
-Tier-Mark : renders
-kimarie-bonuses : basic 'Hello world', renders ok
-freddie-individual-item-details : renders
+Tier-Mark : renders, updates with url change
+kimarie-bonuses : renders with errors
+freddie-individual-item-details : renders, does not change
 
 # Services not working
-max-descriptions : unable to get to run locally, looks like database initialization issues?
+max-descriptions : getting DOM errors
```
Comment (gektin) [inline README.md:15]: Think this is sorted out with the latest merged commit.

**Review by gektin — APPROVED** [review]
All this makes sense. A few comments specific to my service in line, as well as something to consider.

File: README.md (hunk 3 above)
Comment (theofficetroll) [reply README.md:15]: Yeah, I need to update the README on the next pass.

File: server/index.js (hunk 2 above)
Comment (theofficetroll) [reply server/index.js:50]: I probably don't need it, but since I don't have that service, I just left it for now.

File: README.md (hunk 1 above)
Comment (theofficetroll) [reply README.md:6]: Danke

**Pull request merged**